scripts/test/hyriseServer_test.py

```diff
@@ -42,7 +42,7 @@ def main():
 
     client.sendline("COPY loaded_table_from_tbl FROM 'resources/test_data/tbl/int.tbl';")
     client.expect_exact("SELECT 0")
-    client.sendline("SELECT COUNT(*) AS \"row_count\" FROM loaded_table_from_tbl;")
+    client.sendline('SELECT COUNT(*) AS "row_count" FROM loaded_table_from_tbl;')
```

Comment:
It could be an idea to change the other statements to single-quotes, too.
But I think it's a matter of taste.

Reply:
Nevermind, I just saw the format script is doing this.

Reply:
Black prefers double-quote strings (and changes strings to use them). However, black 22.10 on my machine changed it to this version.